### PR TITLE
feat: content_cid not id is used from nft data upload not uploads

### PR DESCRIPTION
**Pull request by dashcraft**
We had been using ID but id is not accurate, content_cid is the id that relates nft uploads to uploads.

### CODE REVIEW

File: packages/api/src/utils/db-client.js [hunk 1]
```diff
@@ -125,20 +125,20 @@ export class DBClient {
...
     }
 
     if (!upload) {
       throw new Error(
-        `Cannot update upload ${JSON.stringify(data.id)} ${JSON.stringify(
-          status
-        )}`
+        `Cannot update upload ${JSON.stringify(
+          data.content_cid
+        )} ${JSON.stringify(status)}`
```
Comment (alanshaw) [inline packages/api/src/utils/db-client.js:141]: Why are these strings/numbers being passed to `JSON.stringify`?

File: packages/api/src/routes/nfts-upload.js [hunk 2]
```diff
@@ -172,15 +172,20 @@ export async function nftUpdateUpload(event, ctx) {
...
-    const id = params.id
+    const content_cid = params.id
 
     // id is required for updating
-    if (!id) return new JSONResponse({ ok: false, value: 'ID is required' })
+    if (!content_cid)
+      return new JSONResponse({ ok: false, value: 'ID is required' })
 
     const body = await event.request.json()
     const { name } = body
```
Comment (alanshaw) [inline packages/api/src/routes/nfts-upload.js:182]: Can we please validate this is a string?

File: packages/api/src/routes/nfts-upload.js [hunk 3]
```diff
@@ -172,15 +172,20 @@ export async function nftUpdateUpload(event, ctx) {
   const { params, db } = ctx
   try {
     const { user } = await validate(event, ctx)
-    const id = params.id
+    const content_cid = params.id
 
     // id is required for updating
-    if (!id) return new JSONResponse({ ok: false, value: 'ID is required' })
+    if (!content_cid)
+      return new JSONResponse({ ok: false, value: 'ID is required' })
```
Comment (alanshaw) [inline packages/api/src/routes/nfts-upload.js:179]:
```suggestion
      return new JSONResponse({ ok: false, value: 'upload CID is required' })
```

**Review by alanshaw — CHANGES_REQUESTED** [review]

File: packages/api/src/utils/db-client.js (hunk 1 above)
Comment (dashcraft) [reply packages/api/src/utils/db-client.js:141]: Leftover cleanup, good catch!

**Review by alanshaw — APPROVED** [review]

**Pull request merged**